ChatQnA/docker_compose/intel/hpu/gaudi/compose_guardrails.yaml

```diff
@@ -25,9 +25,9 @@ services:
       INDEX_NAME: ${INDEX_NAME}
       TEI_ENDPOINT: http://tei-embedding-service:80
       HUGGINGFACEHUB_API_TOKEN: ${HUGGINGFACEHUB_API_TOKEN}
-  tgi-guardrails-service:
-    image: ghcr.io/huggingface/tgi-gaudi:2.3.1
-    container_name: tgi-guardrails-server
+  vllm-guardrails-service:
+    image: ${REGISTRY:-opea}/vllm-gaudi:${TAG:-latest}
+    container_name: vllm-guardrails-server
```

Comment:
I believe you need to change the test scripts for guardrails too. There're quite some tgi related things needs to be refactored